cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityStatus.java

```diff
@@ -74,71 +74,81 @@
 
 /**
  * @author zhangsa
- *
  */
-public class AvailabilityStatus
-{
+public class AvailabilityStatus {
     private boolean _available;
     private Date _upSince;
     private Date _downAt;
     private Date _backAt;
     private String _note;
+    private String _clientIP;
```

Comment:
I wasn't very clear about this when we first talked, but when we made the decision to not put the ipAddress info in the XML doc, it implied that it should not go in the model either.  So, the only place (on the server side) where there is a reference to clientIP is in the toXmlDocument() method in class Availability.  It should be removed from this class and from the servlet.